extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.css

```diff
@@ -133,12 +133,14 @@
     height: 100px !important;
   }
 
-  div.amp-lbg-button-close,
-  div.amp-lbg-button-gallery,
-  div.amp-lbg-button-slide {
+  div.i-amphtml-lbg-gallery {
```

Comment:
nit, general CSS style: unless the div is needed to increase specificity, you can drop it. It can make overriding the rule's properties weird later when you just want to specify a single class for the overriding selector.

![Drop the "div". It's cleaner](https://user-images.githubusercontent.com/2363700/35836233-0affd66e-0a94-11e8-9495-77fc271849dc.jpg)

Reply:
Lol. It's in a media query and therefore needed for the specificity...